src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User user: users) {
+            if (user.getEmail() != null && user.getEmail().equals(email)) {
```

Comment:
I would say here check if user email is null is kinda not needed, however it could be a case for some other inputs.